lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -3135,6 +3135,25 @@ class DecomposeAtenVarMeanCorrectionOp
 };
 } // namespace
 
+namespace {
+// Decompose `aten.randn.generator` op into `aten.randn` op.
+class DecomposeAtenRandnGeneratorOp
```

Comment:
I think we should flip this. `AtenRandnOp` is a special case of `AtenRandnGeneratorOp`, so it makes more sense to decompose the special case to the general case. This will simplify things if we ever add support for generators